### PR TITLE
User story 20 - User Can Edit their Profile Data

### PR DESCRIPTION
**Pull request by tylertomlinson**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [x] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [x] What stories does the PR include

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** = 99.8% 
--------------------------------------------------------------------------------
### Change Log

Created test for editing profile with edge cases and flash messages 
User edit view page 
Created methods `edit` and `update` on the user controller

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2.1
+
+orbs:
+  heroku: circleci/heroku@0.0.10 # Invoke the Heroku orb
```
Comment (danielpselph) [inline .circleci/config.yml]: Awesome work on getting continuous integration implemented

**Review by danielpselph — APPROVED** [review]
Code looks good, may need to refactor for helper methods down the line

**Review by Mariana-21 — APPROVED** [review]
Super robust testing really great!
